legal-api/tests/unit/services/test_authorization.py

```diff
@@ -159,6 +160,7 @@ class FilingKey(str, Enum):
                                     'name': 'registrarsNotation'},
     FilingKey.REGISTRARS_ORDER: {'displayName': "Registrar's Order", 'feeCode': 'NOFEE', 'name': 'registrarsOrder'},
     FilingKey.SPECIAL_RESOLUTION: {'displayName': 'Special Resolution', 'feeCode': 'SPRLN', 'name': 'specialResolution'},
+    FilingKey.AGM_LOCATION_CHANGE: {'displayName': '', 'feeCode': '', 'name': ''},
     FilingKey.ALTERATION: {'displayName': 'Alteration', 'feeCode': 'ALTER', 'name': 'alteration'},
```

Comment:
Also waiting on https://github.com/bcgov/entity/issues/17940 to set and test the expected filing data.